server/src/test/java/io/druid/server/coordinator/DruidCoordinatorTest.java

```diff
@@ -356,4 +375,18 @@ public void childEvent(
     EasyMock.verify(serverInventoryView);
     EasyMock.verify(metadataRuleManager);
   }
+
+  private static void waitFor(Callable<Boolean> condition, long timeout, long interval) throws Exception
+  {
+    assert timeout > 0 && interval > 0;
```

Comment:
i don't think asserts are checked in druid test runs, if you are really uncerstain about timeout and interval then you would use Preconditions. However, given that this method is private , I wouldn't worry about the checks.